functions/Set-DbaAgentJobOutputFile.ps1

```diff
@@ -0,0 +1,139 @@
+﻿
+Function Set-DbaAgentJobOutPutFile
```

Comment:
but please rename to Set-DbaJobOutputFile - both the files and the commands and the tests